### PR TITLE
[fix] rephrase weird chinese translation

### PR DESCRIPTION
**Pull request by theblackcat102**
Some of the chinese localization was unreadable or confusing, even as a chinese native speaker I have trouble understanding the meaning without referring to the english UI

#2417

### CODE REVIEW

File: website/public/locales/zh/error.json
Comment (dddvanming) [inline website/public/locales/zh/error.json:1]: "default": "偶不，阿伯出事了",
Translation Error - the translation does not convey the same meaning as the original sentence.
Change it to, "default": "出了点问题！"

File: website/public/locales/zh/chat.json
Comment (dddvanming) [inline website/public/locales/zh/chat.json:1]: line11:"您的消息已排队，您当前排在第 {{ queuePosition, number, integer }} 位。" is better

File: website/public/locales/zh/index.json
Comment (dddvanming) [inline website/public/locales/zh/index.json:1]: line13:
"a3": "是的，训练数据将以 CC BY 4.0 版权公开。"

Comment (dddvanming) [inline website/public/locales/zh/index.json:1]: line11:
"这些代码和模型是根据Apache 2.0许可证授权的。"
code translate to 代码,程式 is not often use in zh.

**Review by notmd — APPROVED** [review]
@theblackcat102 Please merge when you feel it's ready..

File: website/public/locales/zh/index.json
Comment (theblackcat102) [reply website/public/locales/zh/index.json:1]: depends on region I guess

File: website/public/locales/zh/error.json
Comment (theblackcat102) [reply website/public/locales/zh/error.json:1]: sure， I remove the ! to make the meaning more neutral, same as other message

File: website/public/locales/zh/chat.json
Comment (theblackcat102) [reply website/public/locales/zh/chat.json:1]: your translation is way too strict, do you say "你的回覆正在排隊中"?

File: website/public/locales/zh/index.json
Comment (theblackcat102) [reply website/public/locales/zh/index.json:1]: aren't they the same meaning?

**Comment by theblackcat102** [comment]
@dddvanming any idea on the rest of the translation in `stats.json`? I haven't translated these yet

File: website/public/locales/zh/index.json
Comment (dddvanming) [reply website/public/locales/zh/index.json:1]: Yes,the term "程式" is primarily used in Hong Kong, Macau, and Taiwan, while "程序" is more commonly used in mainland China.

Comment (dddvanming) [reply website/public/locales/zh/index.json:1]: Traditional Chinese characters are predominantly used in Hong Kong, Macau, and Taiwan, while simplified Chinese characters are used in mainland China.

File: website/public/locales/zh/chat.json
Comment (dddvanming) [reply website/public/locales/zh/chat.json:1]: In my personal estimation, queuing is a more intuitive approach.

File: website/public/locales/zh/index.json
Comment (dddvanming) [reply website/public/locales/zh/index.json:1]: Yes, but the term "数据" is commonly used in computer science and represents information that is stored in digital form. In computer science, data can take the form of text, images, audio, video, or any other type of digital information. Data can be processed and manipulated by computers, including tasks such as storage, sorting, searching, filtering, conversion, and analysis. Therefore, the term "数据" is more appropriate for use in the field of computer science

**Comment by dddvanming** [comment]
> @dddvanming any idea on the rest of the translation in `stats.json`? I haven't translated these yet

If the meanings of those message tree states are known, the translation will be more accessible.

**Comment by dddvanming** [comment]
> @dddvanming any idea on the rest of the translation in `stats.json`? I haven't translated these yet

If the meanings of those message tree states are known, the translation will be more accessible.

File: website/public/locales/zh/index.json
Comment (dddvanming) [reply website/public/locales/zh/index.json:1]: and the i18n file is simplified Chinese,so we should follow the the habits of people who use Simplified Chinese.

Comment (theblackcat102) [reply website/public/locales/zh/index.json:1]: hmm, fine

**Pull request merged**